assets/README.md

```diff
@@ -0,0 +1,5 @@
+## Gutenberg Plugin Assets
+
+The contents of this directory are synced to the [`assets/` directory of the Gutenberg plugin](https://plugins.trac.wordpress.org/browser/gutenberg/assets) in the WordPress.org plugin repository. This is done by a [GitHub Actions workflow](https://github.com/WordPress/gutenberg/actions/workflows/sync-assets-to-plugin-repo.yml) that is triggered whenever a file in this directory is changed (with the exception of this README, which is excluded from the sync).
```

Comment:
```suggestion
This directory is managed via the [`assets/` directory in the Gutenberg repository on GitHub](https://github.com/WordPress/gutenberg/tree/trunk/assets). **Changes committed directly to the plugin repository on WordPress.org will be overwritten.**

The contents of this directory are synced with the [`assets/` directory of the Gutenberg WordPress.org plugin repository](https://plugins.trac.wordpress.org/browser/gutenberg/assets). This is done by a [GitHub Actions workflow](https://github.com/WordPress/gutenberg/actions/workflows/sync-assets-to-plugin-repo.yml) that is triggered whenever a file in this directory is changed.
```

Reply:
Applied. 

I think I'll shuffle the paragraphs a bit though. The first paragraph currently makes it easier to understand when read in the plugins repo, but potentially a bit harder when read in the GH repo, which I think will be the more common case.